eZ/Publish/API/Repository/Tests/RoleServiceTest.php

```diff
@@ -613,6 +959,47 @@ public function testAddPolicyUpdatesRole()
     }
 
     /**
+     * Test for the addPolicyByRoleDraft() method.
+     *
+     * @return array [\eZ\Publish\API\Repository\Values\User\RoleDraft, \eZ\Publish\API\Repository\Values\User\Policy]
+     *
+     * @see \eZ\Publish\API\Repository\RoleService::addPolicyByRoleDraft()
+     * @depends eZ\Publish\API\Repository\Tests\RoleServiceTest::testAddPolicyByRoleDraft
+     */
+    public function testAddPolicyByRoleDraftUpdatesRole()
+    {
+        $repository = $this->getRepository();
+
+        /* BEGIN: Use Case */
+        $roleService = $repository->getRoleService();
+
+        $roleCreate = $roleService->newRoleCreateStruct('newRole');
+
+        // @todo uncomment when support for multilingual names and descriptions is added
```

Comment:
Maybe we should include an issue # in the comment, so that we can easily find those ?